### ISSUE TITLE
[49.1+390] In MHQ, all units appear with ID:-1

### ISSUE THREAD

**Issue by Tecmes** [MegaMek/mekhq#2613]
When checking the units in a pending scenario they all have ID-1 (except your own).  It matters little, as they are correct in MM (in fact I wonder why there's an ID shown at all in MHQ?).
To reproduce, check any scenario.

**Comment by sixlettervariables** [comment]
If you are using IDs and the units have not been through MM, they will not have IDs. I haven't confirmed in the code, so leaving this open for now. (I believe I investigated this before).

**Comment by NickAragua** [comment]
One comment I'll need to make is that those unit IDs are transient, arbitrary and not necessarily persistent between MekHQ sessions. I don't think this is an issue that needs addressing (except maybe to hide said unit IDs in the MekHQ interface).

**Comment by Tecmes** [comment]
Yeah, hiding them is really what I was meaning. UNless I'm missing something, while the ID are quite useful in the lobby, they don't seem to have any use in the MHQ scenario pane.

**Comment by IllianiCBT** [comment]
Closing based on Nick's comments.
